EventOrganizer/src/EventOrganizer.Blazor/Pages/CreateEvent.razor

```diff
@@ -3,27 +3,29 @@
 @using EventOrganizer.Localization
 @inject IStringLocalizer<EventOrganizerResource> L
 
-<Heading Size="HeadingSize.Is3" Class="text-center">Create Event</Heading>
+<Heading Size="HeadingSize.Is3" Class="text-center mt-5 mb-4">Create Event</Heading>
```

Comment:
Instead of using hardcoded classes, it's prefered to use Blazorise utilities. For this, it should be `Margin="Margin.Is5.FromTop.Is4.FromBottom"`